api/api-doc/storage_service.json

```diff
@@ -2864,6 +2864,14 @@
                      "allowMultiple":false,
                      "type":"string",
                      "paramType":"query"
+                  },
+                  {
+                     "name":"await_completion",
+                     "description":"Set true to wait for the repair to complete. Set false to skip waiting for the repair to complete. When the option is not provided, it is default to true.",
```

Comment:
How can you have a default when this is a required param? Indeed don't see a default anywhere below.

Reply:
It is not a required param, see:

```
"required":false,
```

Reply:
Right, I don't know how I interpreted `false` as required. Anyway, I think `false` is a better default -- we are moving to an async by default API.
Actually, is there any reason to even make this configurable? We now have task api to wait for repair? Is there a release where this is the only way to wait for the tablet repair, that is already released and we need to be backward compatible with?

Reply:
The wait does not depend on the virtual task stuff which we have observed issue (task manager reports no such task while the task is present in system.tablets). I want to keep the wait functionality in this api. 
Another reason is that to be compatible with existing behavior of the api. We do not have a release yet but we have tests that use it. Even if we want to change the default to 'not wait', we'd better do it as a follow up to adjust all the usages separately.

Reply:
Ok, I'm fine with keeping the option, but please let's make the default be false by the time we release, so this becomes an async-by-default API. We want to make all our APIs that start long-term work async, adding a new sync-by-default API is working against that goal.

Reply:
I looked the tests. We can actually switch to default false without touching them. I pushed a new version with default false.